### ISSUE TITLE
Custom (Accept) headers via .yml config file

### ISSUE THREAD

**Issue by aleksblendwerk** [calliostro/discogs-bundle#1]
First of all: thanks for your work on the great lib and the accompanying bundle, @calliostro!

I just installed this bundle to make quick and easy usage of the actual library in a Symfony project but went back to including your library without this bundle for the simple reason that I can't find an easy way to set the `Accept` header via the configuration files. (If there is one, I'll stand corrected...)

I want to set a custom header because endpoints like https://api.discogs.com/labels/9291 deliver a response where the field `profile` contains Discogs' special BBCode-ish dialect - and I am not in the mood to parse this right now.

As soon as you specify an `Accept` header with `application/vnd.discogs.v2.html+json`, the Discogs API will return an additional field called `profile_html` that contains - well - HTML.

Described here: https://www.discogs.com/developers/#page:home,header:home-versioning-and-media-types

So tl;dr: it would be great if there was an easy way to set a custom `Accept` header (or maybe even other headers too) via the bundle's `.yml` configuration.

**Comment by calliostro** [comment]
Hi @aleksblendwerk,

This is now fully supported in v4.0.0! You can set custom headers directly in the configuration:

```yaml
# config/packages/calliostro_discogs.yaml
calliostro_discogs:
    user_agent: 'MyApp/1.0 +https://myapp.com'
    http_client:
        headers:
            Accept: 'application/vnd.discogs.v2.html+json'
```

**Installation:**
```bash
composer require calliostro/discogs-bundle:^4.0
```

The v4.0 rewrite brings clean parameter API, full Guzzle configuration support, and compatibility with PHP 8.1-8.5 and Symfony 6.4-8.x.

See [README](https://github.com/calliostro/discogs-bundle#readme) for details. Let me know if this solves your use case!

**Comment by calliostro** [comment]
Closing this as the feature is now implemented in v4.0.0. Feel free to reopen if you have any issues with the new version!
